examples/proxy/record-encryption/README.md

```diff
@@ -1,20 +1,17 @@
 # Streams for Apache Kafka Proxy Record Encryption Filter Examples
 
-The Record Encryption filter provides an Encryption-at-Rest solution for Apache Kafka(tm) which is transparent to both clients and brokers. The filter takes
-responsibilty to encrypt messages that are sent by producing applications so the Kafka Broker never sees the plain text content of the messages.  The filter also takes responsibilty
-to decrypt the message before they are returned to consuming applications.
+The Record Encryption filter provides an Encryption-at-Rest solution for Apache Kafka(tm) which is transparent to both clients and brokers. 
+The filter is responsible for encrypting messages that are sent by producing applications so the Kafka Broker never sees the plain text content of the messages.  
+The filter also decrypts messages before they are returned to consuming applications.
```

Comment:
If @PaulRMellor is happy with the phrasing then I guess I should shut-up but...

I find it odd to say the `filter is responsible for` on the produce path but only `also decrypts` for the fetch path.  Though I think it does read better than the previous version.

Reply:
@PaulRMellor wdyt?  "is responsible for encrypting " => "encrypts" is less wordy.